apps/judicial-system/web/src/utils/utils.ts

```diff
@@ -147,3 +147,30 @@ export const shouldUseAppealWithdrawnRoutes = (theCase: Case): boolean => {
       !theCase.appealJudge3)
   )
 }
+
+/**
+ * Returns true if all subarrays in the given object have only one item.
+ *
+ * Example:
+ * const indictmentSubtypes = {
+ *   '123-123-123': ['item1'],
+ *   '234-234-234': ['item2']
+ * }
+ *
+ * const result = hasOnlyOneItemInSubArrays(indictmentSubtypes)
+ * console.log(result) // true
+ *
+ * @param indictmentSubtypes - The object containing the subarrays to check.
+ * @returns {boolean} - True if all subarrays have only one item, false otherwise.
+ */
+export const hasOnlyOneItemInSubArrays = (
+  indictmentSubtypes?: Record<string, string[]>,
```

Comment:
Consider naming this something other than indictmentSubtypes as the rest of the function seems to be more general, so we could theoretically use it to check this for other subarrays that aren't indictment subtypes

Reply:
Yes, thanks. I've renamed the parameter and some other properties of this function. Let me know if anything is unclear.